source/deployment-options/elastic-stack/distributed-deployment/elasticsearch-cluster/elasticsearch-multi-node-cluster.rst

```diff
@@ -172,7 +172,7 @@ Certificates deployment
 
     .. include:: /_templates/installations/basic/elastic/common/elastic-multi-node/deploy_certificates_subsequent_nodes.rst
 
-#. If Kibana will be installed in this node, keep the certificates file. Otherwise, remove it to increase security ``rm -f ~/certs.zip``.  
+#. If Kibana is installed in this node, keep the certificates file. Otherwise, remove it to increase security ``rm -f ~/certs.zip``.  
```

Comment:
```suggestion
#. If you are going to install Kibana in this node, keep the certificates file. Otherwise, remove it to increase security ``rm -f ~/certs.zip``. 
```